scene/3d/gi_probe.cpp

```diff
@@ -493,7 +493,7 @@ void GIProbe::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("debug_bake"), &GIProbe::_debug_bake);
 	ClassDB::set_method_flags(get_class_static(), _scs_create("debug_bake"), METHOD_FLAGS_DEFAULT | METHOD_FLAG_EDITOR);
 
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "subdiv", PROPERTY_HINT_ENUM, "64,128,256,512"), "set_subdiv", "get_subdiv");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "subdiv", PROPERTY_HINT_ENUM, "64 (Fast),128 (Average),256 (Slow),512 (Slowest)"), "set_subdiv", "get_subdiv");
```

Comment:
I used "Slowest" for 512 instead of "Slower" because it's *very* slow in `3.x` compared to `master`.